GoogleUtilities/Environment/NetworkInfo/GULNetworkInfo.m

```diff
@@ -70,13 +70,11 @@ + (GULNetworkType)getNetworkType {
 + (NSString *)getNetworkRadioType {
 #ifdef TARGET_HAS_MOBILE_CONNECTIVITY
   CTTelephonyNetworkInfo *networkInfo = [GULNetworkInfo getNetworkInfo];
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-  return networkInfo.currentRadioAccessTechnology;
-#pragma clang diagnostic pop
-#else
-  return @"";
+  if (networkInfo.serviceCurrentRadioAccessTechnology.count) {
+    return networkInfo.serviceCurrentRadioAccessTechnology.allValues[0] ?: @"";
```

Comment:
`serviceCurrentRadioAccessTechnology` has iOS 12+ availability. `allValues[0]` should return a String, but since Sessions uses this from Swift, I'm adding the `?: @""` to prevent a bridging crash if a non-string/corrupted value is returned.